piplicenses.py

```diff
@@ -152,6 +152,19 @@ def normalize_pkg_name(pkg_name: str) -> str:
     return PATTERN_DELIMITER.sub("-", pkg_name).lower()
```

Comment:
Stabilize with [strip()](https://docs.python.org/3/library/stdtypes.html#str.strip)
```suggestion
    return PATTERN_DELIMITER.sub("-", pkg_name).lower().strip()
```